ISISComputingGroup/ibex_gui
# Ticket967 populate opi property options

ISISComputingGroup/IBEX/issues/967

To test:
* Select a component target with no properties, such as Caen HV. Check the properties table and value boxes are disabled.
* Select a component target with properties, such as Pinhole Selector. Check the properties are populated appropriately, values can be set and descriptions provided.
* Check everything behaves as expected when switching between components.
* Check everything behaves as expected when changing the target.
* Check saving and loading the synoptic behaves correctly (you can check either the saved XML too).
* Check old synoptics have properties set correctly, e.g. slits on Larmor.
* Check synoptic get macros set as before.
